### DISCUSSION TITLE
old automation history clutters export

### DISCUSSION THREAD

**Discussion by yeoldegrove** [Budibase/budibase#10430] [Help] [answered]
When I export my app (without the rows) and have a look at the ~100MB `db.txt` in the tar file, I see that most of the data is indeed from old automation history. I had some very verbose logging in the automatons a while ago.

I think a switch in the export UI would be nice to disable the export of these old logs.
The unpaid version cannot display the old logs anyway.

Also, is there a nice way to filter out these old logs from the command-line and re-import the app without the old clutter?
I sadly cannot share the app, as there is way too much sensitive information in the old automation logs.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @yeoldegrove,

We are going to remove automation logs completely from app exports as they aren't needed.
This change should be available soon.

The PR can be found here: https://github.com/Budibase/budibase/pull/10432 

I hope this resolves the issue!

**Reply by yeoldegrove** [reply]
Thanks for the swift fix @ConorWebb96 and @mike12345567 !
When will the next release come out, so that I can test it?
